flow-defs/list.js.flow

```diff
@@ -28,10 +28,12 @@ declare type BasicRowContentProps = {|
 |};
 declare type SwitchRowContentProps = {|
   ...$Exact<CommonProps>,
+  onPress?: () => void,
```

Comment:
1. Doesn't works with `href` prop. Is it expected?

https://mistica-web-pladaria.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AlCB3AMgSwM4BcA%2BAHQDsACc1DMyy7SUgXmFrsqQGFsAnMAGxjkARgEMwAawDmPCAFdSUThH4QeLSKp64AdKRhz8PUf0wYAvuVzYAXjBYAWAAzmSFdnSQBJRgGUAFhAADigwUnL8ojzkAPRuHlQx3HyC8XTmbPR%2BtvbAzhnulPjY%2BIJMxCAAKiWCFZnksLhgPNhBxRDMFQAiME0tbdgddYXk-jwwAGblIP74%2BEG48DEx6Ks6UhAQUoI6vcPsHQAK47i4LAAUAJTkTITkwOYF7LjoJWD%2BLMANk6IR%2BABqJjkMHg5AmJlwQkebDibGo6HqDA6LHqHGSAiEYkkMnkimUWg0KjUun0hmMpgsVhyjhcaQS3j8gRCYQiUVidM8SV4GI5TzoSNIvmpeRc9WKpXsFWqEv2dEazVa7U6IB6fUVg1IssoRxOZ2AVxudwefMoLzeH2AX1g4L%2BgP4wNB4P4kPI0PcsNISBiaCweCIZBA5iAA

2. Only works for switches? Should we have the same behavior to radio controls?

Reply:
This only works with switch/checkbox + onPress. 

We don't have any other cases in our code, if needed we could implement it... but if not needed I prefer to not complicate the code.